packages/app/src/app/pages/Sandbox/Editor/Workspace/ConnectionNotice/index.ts

```diff
@@ -0,0 +1 @@
+export { ConnectionNotice } from './ConnectionNotice';
```

Comment:
@Saeris Why cant we just delete this file and rename `ConnectionNotice.jsx` to `index.tsx`? 🤔

Right now we're creating every time an extra file for no reason at all imo.

Reply:
Two reasons: readability in the editor (every open tab becomes index.ts otherwise), and having a quick reference for what a directory exports.

Reply:
Sounds reasonable to me

We're not doing this anywhere else in the code, so I think @CompuIves is the best man to decide on this one

Reply:
As long as we're going to do this consistently in the future I'm okay with this!